packages/libsql-client/src/http.ts

```diff
@@ -78,7 +78,6 @@ export class HttpClient implements Client {
     protocol: "http";
     #url: URL;
```

Comment:
We don't need to keep `#url` nor `#authToken` as instance variables because they were being used only for the migrations related functions. You can safely delete both of them here and from the constructor assignment.

<img width="771" alt="image" src="https://github.com/tursodatabase/libsql-client-ts/assets/1259399/66779c30-a183-4f23-9a9c-f0f72470bee6">